packages/react-badge/src/components/Badge/useBadge.ts

```diff
@@ -26,7 +26,6 @@ export const useBadge_unstable = (props: BadgeProps, ref: React.Ref<HTMLElement>
     },
     root: getNativeElementProps('div', {
       ref,
-      'aria-hidden': true,
```

Comment:
does this also change accessibility / DOM of avatar?

Reply:
Yes it does. I believe @smhigley wanted that hidden removed from all badges. I'll confirm.

Reply:
yup, I don't think Badge should've been hidden, since it often contains meaningful text or graphics. And when it doesn't, it shouldn't be exposed even without `aria-hidden` :).

I think exposing its content is a much safer default than hiding it.

Reply:
in that case the best practices need to be updated:

<img width="996" alt="image" src="https://user-images.githubusercontent.com/13742664/166725829-de6e6eb2-310f-4c7a-b404-b09f9438f7dd.png">

Reply:
change moved to https://github.com/microsoft/fluentui/pull/22858